### PR TITLE
Update access-management.md

### PR DESCRIPTION
**Pull request by yanivhershkovich**
Based on a recent query from a customer and after doing a few tests, suggesting changes to the 'GCP Marketplace team management' section.
I suggest having these changes reviewed by PM and/or R&D to confirm them.

### CODE REVIEW

File: content/rc/administration/access-management.md
Comment (lanceleonard) [inline content/rc/administration/access-management.md:133]:
```suggestion
- A user with a 'Browser' role cannot access the Redis admin console.
```

File: content/rc/administration/access-management.md
Comment (lanceleonard) [inline content/rc/administration/access-management.md:132]:
```suggestion
- A 'Viewer' role lets the user access the Redis admin console but not make changes.
```

File: content/rc/administration/access-management.md
Comment (lanceleonard) [inline content/rc/administration/access-management.md:131]:
```suggestion
- 'Owner' and 'Editor' roles let users access the Redis admin console and make changes.
```

**Review by lanceleonard — APPROVED** [review]
LGTM.  Provisionally approved (with edits & pending PM approval).

**Comment by rrelledge** [comment]
Closing this PR without merging as this info is now outdated. PR #2324 removes this section since it is no longer needed for the latest GCP Marketplace listing.

**Pull request closed without merging**
